functions/Copy-DbaAgentAlert.ps1

```diff
@@ -150,6 +150,28 @@ function Copy-DbaAgentAlert {
                     continue
                 }
 
+                $operatorDoesntExist = $false
+                $sourceAlertNotifications = Get-DbaAgentAlert -SqlInstance $sourceServer.Name | Where-Object Name -eq $serverAlert.name | Select-Object Notifications
+                foreach ($sourceAlertNotification in $sourceAlertNotifications)
+                {
+                 $operatorName = $sourceAlertNotification.Notifications.OperatorName
+                 if (-Not(Get-DbaAgentOperator -SqlInstance $destinstance | Where-Object Name -eq $operatorName))
+                 {
+                  $operatorDoesntExist = $true
+                  if ($PSCmdlet.ShouldProcess($destinstance, "Operator [$operatorName] does not exist at destination.")) {
+                      $copyAgentAlertStatus.Status = "Skipped"
+                      $copyAgentAlertStatus.Notes = "Operator [$operatorName] doesn't exist on destination"
+                      $copyAgentAlertStatus | Select-DefaultView -Property DateTime, SourceServer, DestinationServer, Name, Type, Status, Notes -TypeName MigrationObject
+                      Write-Message -Message "Operator [$operatorName] does not exist at destination." -Level Verbose
+                  }
+                 }
+                }
+                if ($operatorDoesntExist)
```

Comment:
So something like 

```
if ($operatorNotExists) {
    if ($PSCmdlet.ShouldProcess....) {
        $copyAgentAlertSTatus....
        ...
        Write-Message -Message "Operator []..." -Level Warning
        continue
    }
}
```

`Stop-Function` should be used in try/catch blocks and not with if statements. You can simply just output a warning that the alert is being skipped because the operator does not exist, and then continue on.

Reply:
Small note I'd rather use `operatorNotExists` than use contractions in variable names (just reads better).

Reply:
Thanks, @wsmelton . Fixed and committed along with incorporating the approach by @alevyinroc to get the list of operators (with a small correction found while testing). Thanks, @alevyinroc!